dojo/tools/h1/parser.py

```diff
@@ -70,11 +82,17 @@ def get_findings(self, file, test):
 
             # Set CWE of the Dojo finding
             try:
-                cwe = int(content["relationships"]["weakness"]["data"]["attributes"]["external_id"][4:])
-            except:
+                cwe = int(
+                    content["relationships"]["weakness"]["data"]["attributes"][
+                        "external_id"
+                    ][4:]
+                )
+            except BaseException:
```

Comment:
```suggestion
            except Exception:
```

Reply:
Changes apply in commit https://github.com/DefectDojo/django-DefectDojo/pull/8302/commits/7a4f4778c0bda57e031626cb1dece9dfabb8bf50